lms/routes.py

```diff
@@ -223,3 +223,10 @@ def includeme(config):  # pylint:disable=too-many-statements
     config.add_route(
         "lti.v11.deep_linking.form_fields", "/lti/1.1/deep_linking/form_fields"
     )
+
+    # We need boh to apply different security policies to them.
+    # We could go around that implementation our own schema location `form_or_cookie`
```

Comment:
We could also redo security.py a bit so we can apply route + method to policies.